README.md

```diff
@@ -13,14 +13,17 @@ A curated list of awesome things and projects built by Algerian developpers.
 ## Open Data
 - [Algerian Administrative Division](https://github.com/mohsenuss91/AlgerianAdministrativeDivision) - Wilayas - Daïras - baladiyahs
 - [Algerial Logos](https://github.com/mohsenuss91/AlgerianLogos) - Some Algerian official vectorized (.SVG) Logos
+- [algeria-cities](https://github.com/othmanus/algeria-cities) - A complete list of Algeria provinces and municipalities in csv, xlsx, sql, php and json formats.
 
 ## Applications
 - [Alfanous](https://github.com/Alfanous-team/alfanous) - Alfanous is an Arabic search engine API provides the simple and advanced search in Quran
+- [DzLibs](https://github.com/01walid/dzlibs) - A community-driven Algerian index of reusable assets and libraries.
```

Comment:
I like this project but It's not active for more than 2 years, and I think we should keep only active projects (see the rules on the README).